.github/workflows/dotnet.yaml

```diff
@@ -8,32 +8,32 @@ jobs:
         os: [ubuntu-latest, windows-latest, macOS-latest]
     name: Dotnet build
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@master
         with:
           fetch-depth: 0
       - name: Setup dotnet runtime 2.0.9
         uses: actions/setup-dotnet@v1
         with:
           dotnet-version: 2.1.202
-      - name: Setup dotnet runtime 2.1.17
+      - name: Setup dotnet runtime 2.1.0
         uses: actions/setup-dotnet@v1
         with:
-          dotnet-version: 2.1.805
+          dotnet-version: 2.1.300
```

Comment:
Why move backward in version-space?

Reply:
We actually have an indirect dependency on 2.1.0 in tests project which started throwing when strong signing was enabled on test project.

Reply:
@macsux 2.1.805 is the latest 2.1 SDK release and it would be preferable to use the bundled runtime.

> when strong signing was enabled on test project.

Can you elaborate on the error? Is it necessary to sign the tests?

Reply:
Strong signing for tests was enabled to allow unit testing internal classes/methods. In order to make test assembly "friend", it had to be strongly signed because main assembly is strongly typed. I don't remember the error now, but it was looking for specific version of some package that come with .net core 2.1.0. It was only showing up on the build server as I have every version of .NET on my dev machine.

Reply:
Understood (I had yet to reach that change).

I see the [build](https://github.com/kubernetes-client/csharp/actions/runs/73778750) for 2.1.805 succeed anyway.

Reply:
Ok, I reverted it to your version, not sure why but it works now. Gonna write off the error on something weird I was doing at the time.